portfolio/src/main/java/com/google/sps/data/BasketballCourt.java

```diff
@@ -0,0 +1,27 @@
+package com.google.sps.data;
+
+import java.util.OptionalInt;
+
+/** Class used to store the json object from the basketball courts file.
+  Doesn't require getter or setter methods because the class is just for the 
+  basketball court to be returned as the api response. numOfCourts not always
+  provided which is why its Optional. */
+public final class BasketballCourt {
+    private final String id;
+    private final String name;
+    private final String location;
+    private final OptionalInt numOfCourts;
+    private final double lat;
+    private final double lon;
+
+    public BasketballCourt(String id, String name, String location,
+      OptionalInt numOfCourts, double lat, double lon) {
```

Comment:
super nit: optional arguments should probably be last in the list of arguments
(and keep them sorted in your member vars and constructor assignments list too)